pages.de/common/clang-tidy.md

```diff
@@ -0,0 +1,20 @@
+# clang-tidy
+
+> Ein LLVM-basierter C/C++ Linter zum Finden von Stilverletzungen, Bugs und Sicherheitsmängeln durch statische Codeanalyse.
+> Weitere Information: <https://clang.llvm.org/extra/clang-tidy/>.
+
+- Führe die Standard-Checks für eine Quelldatei aus:
+
+`clang-tidy {{pfad/zu/quelldatei.cpp}}`
+
+- Prüfe nur ob eine Datei den `cppcoreguidelines` Checks entspricht:
+
+`clang-tidy {{pfad/zu/quelldatei.cpp}} -checks={{-*,cppcoreguidelines-*}}`
```

Comment:
```suggestion
`clang-tidy {{pfad/zu/quelldatei.cpp}} -checks="{{-*,cppcoreguidelines-*}}"`
```

Optional fix (source: [original page](https://github.com/tldr-pages/tldr/blob/main/pages/common/clang-tidy.md)): globbing characters can be quoted to prevent POSIX-compatible shells from treating them as glob patterns.